client/js/views/templates/OurOfferings.js

```diff
@@ -1,3 +1,16 @@
 module.exports = function( { model } ) {
-    return `<div>our offerings</div>`
+	const offerings = model.reduce((memo, offer) => {
```

Comment:
Now you can have two variables, `fallOfferings` and `yearRoundOfferings`:
`const fallOfferings = model.fall.reduce...` and so on.